vnext/src/Libraries/AppTheme/AppTheme.ts

```diff
@@ -40,12 +40,24 @@ class AppThemeModule extends NativeEventEmitter {
     this.addListener(
       'appThemeChanged',
       ({currentTheme}: {currentTheme: AppThemeTypes}) => {
+        warnOnce(
```

Comment:
I think this will fire any time system theme changes, even if we don't have a subscriber.

Reply:
Removing the AutoMerge label quickly.

Would you be able to test the scenario of using a supported API (e.g. subscribing to contrast changed events), then changing system color? I think as is this will potentially warn about the color subscription API being removed even if it isn't used.

Reply:
Thanks for catching this. Should be good now.